netmaster/gstate/gstate.go

```diff
@@ -438,7 +518,18 @@ func (gc *Cfg) UpdateResources(res string) error {
 			log.Errorf("error '%s' updating global oper state %v \n", err, g)
 			return err
 		}
+	} else if res == "epg" {
```

Comment:
This will only be needed if we are updating the resource ranges. Since we will not be doing for epg. should this be ignored ?